server/service/src/purchase_order_line/insert/mod.rs

```diff
@@ -62,23 +77,33 @@ pub fn insert_purchase_order_line(
             let validate_input = validate::ValidateInput {
                 id: input.id.clone(),
                 purchase_order_id: input.purchase_order_id.clone(),
-                item_id: Some(input.item_id.clone()),
-                item_code: None,
-                // TODO amend default value if we extend standard insert line input
-                requested_pack_size: input.requested_pack_size.unwrap_or_default(), // Default value
+                item_id: input.item_id.clone(),
+                item_code: input.item_code.clone(),
+                requested_pack_size: input.requested_pack_size.unwrap_or_default(),
                 manufacturer_id: input.manufacturer_id.clone(),
             };
             let item = validate(&ctx.store_id.clone(), &validate_input, connection)?;
+
+            let generate_input = InsertPurchaseOrderLineInput {
+                item_id: Some(item.id.clone()),
+                item_code: None,
+                ..input
+            };
+
             let new_purchase_order_line =
-                generate(connection, &ctx.store_id.clone(), item, input.clone())?;
+                generate(connection, &ctx.store_id.clone(), item, generate_input)?;
+
+            // Only log activity for standard mode (not CSV bulk imports)
```

Comment:
Why won't there be a long for CSV? Could be a 'bulk created from CSV log'...

Reply:
I was about to ask about that. Before refactor the insert_csv didn't have log. But makes a lot of sense to have it as well